styles.css

```diff
@@ -122,4 +123,955 @@
     padding: 120px 24px;
     gap: 12px;
   }
+
+  .recent-works {
+    display: flex;
+    align-items: flex-start;
+    justify-content: flex-start;
+    flex-direction: column;
+    padding: 25px;
+    gap: 30px;
+    position: absolute;
+    width: 375px;
+    height: 3502px;
+    top: 638.2px;
+    background-color: #e5e5e5;
```

Comment:
- [ ] [OPTIONAL]As stated in the project requirement kindly use CSS Grid in the main container that contains all of the cards 🙏